### PR TITLE
fix(mydevil-deploy): Update mydevil ssh key

### PR DESCRIPTION
**Pull request by k911**
Fixes deploying to test environment (mydevil) using travis by providing new encrypted ssh key

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@ before_script:
 script:
   - yarn run test
   - yarn run lint
-  - yarn run lint:scss
+  - 'yarn run lint:scss'
```
Comment (FieryCod) [inline .travis.yml]: Please consider removing `''`

Comment (k911) [reply .travis.yml]: Apparently VS Code Yaml Formatting considered it dangerous due to ':' character, is it ok? @FieryCod

Comment (FieryCod) [reply .travis.yml]: Let it be like it is if the extension thinks it's dangerous. It worked previously but ok :D Merge when you'll be ready.

**Comment by FieryCod** [comment]
Thank you for the PR! GJ

**Pull request merged**